docs/guide/hello.md

```diff
@@ -124,98 +124,110 @@ For all subsequent commands, use a terminal window that is different from the wi
 
+Note: it's recommended to commit changes to a version control system (for example, Git) after scaffolding. This allows others to easily distinguish between code generated by Ignite and the code writen by hand.
+
+```
+git add .
+git commit -am "Scaffolded a hello query with Ignite CLI"
+```
+
 ### Updates to the query service
 
-In the `proto/hello/query.proto` file, the `Posts` rpc has been added to the `Query` service.
+In the `proto/hello/query.proto` file, the `Hello` rpc has been added to the `Query` service.
 
 ```proto
 service Query {
-  rpc Posts(QueryPostsRequest) returns (QueryPostsResponse) {
-    option (google.api.http).get = "/username/hello/hello/posts";
-  }
+	rpc Hello(QueryHelloRequest) returns (QueryHelloResponse) {
+		option (google.api.http).get = "/hello/hello/hello";
+	}
 }
 ```
 
-Here's how the `Posts` rpc for the `Query` service works:
+Here's how the `Hello` rpc for the `Query` service works:
 
-- Is responsible for returning a list of all the posts on chain
-- Accepts request parameters (`QueryPostsRequest`)
-- Returns response of type `QueryPostsResponse`
+- Is responsible for returning a `text` string
+- Accepts request parameters (`QueryHelloRequest`)
+- Returns response of type `QueryHelloResponse`
 - The `option` defines the endpoint that is used by gRPC to generate an HTTP API
 
 ### Request and reponse types
 
 Now, take a look at the following request and response types:
 
 ```proto
-message QueryPostsRequest {
+message QueryHelloRequest {
 }
 
-message QueryPostsResponse {
-  string title = 1;
-  string body = 2;
+message QueryHelloResponse {
+  string text = 1;
 }
 ```
 
-- The `QueryPostsRequest` message is empty because requesting all posts doesn't require parameters.
-- The `QueryPostsResponse` message contains `title` and `body` that is returned from the chain.
+- The `QueryHelloRequest` message is empty because this request does not require parameters.
+- The `QueryHelloResponse` message contains `text` that is returned from the chain.
 
-## Posts keeper function
+## Hello keeper function
 
-The `x/hello/keeper/grpc_query_posts.go` file contains the `Posts` keeper function that handles the query and returns data.
+The `x/hello/keeper/grpc_query_hello.go` file contains the `Hello` keeper function that handles the query and returns data.
 
 ```go
-func (k Keeper) Posts(c context.Context, req *types.QueryPostsRequest) (*types.QueryPostsResponse, error) {
-  if req == nil {
-    return nil, status.Error(codes.InvalidArgument, "invalid request")
-  }
-  ctx := sdk.UnwrapSDKContext(c)
-  _ = ctx
-  return &types.QueryPostsResponse{}, nil
+func (k Keeper) Hello(goCtx context.Context, req *types.QueryHelloRequest) (*types.QueryHelloResponse, error) {
```

Comment:
Just an idea, if it makes sense `rpcCtx` or `queryCtx` could also be used as names for the context argument within the Keeper methods.

Reply:
Maybe we should consider changing these names in the template (because the tutorial uses whatever the template provides).